draft-packages/badge/KaizenDraft/Badge/styles.module.scss

```
@@ -77,7 +79,7 @@ $small: $spacing-md;
   color: $color-white;
 }
 
-$badge-duration-timing: ($animation-duration-slow / 2)
+$badge-duration-timing: calc($animation-duration-slow / 2)
```

```suggestion
$badge-duration-timing: calc(#{$animation-duration-slow}/ 2)
```
This is a CSS variable, so I believe you'll want to ensure it's interpolated.